activesupport/CHANGELOG.md

```diff
@@ -1,3 +1,19 @@
+*   Fix a regression with division by a Duration object
+
+    Before:
+        rental_end = DateTime.parse('01-01-2017 12:45:00').utc.in_time_zone('Europe/Berlin')
+        rental_start = DateTime.parse('01-01-2017 12:00:00').utc.in_time_zone('Europe/Berlin')
+        (rental_end - rental_start) / 1.hour
+        => 2700.0 hours
+    
+    After:
+        rental_end = DateTime.parse('01-01-2017 12:45:00').utc.in_time_zone('Europe/Berlin')
+        rental_start = DateTime.parse('01-01-2017 12:00:00').utc.in_time_zone('Europe/Berlin')
+        (rental_end - rental_start) / 1.hour
+        => 0.75
```

Comment:
If this change makes the same behavior as Rails 5.1.1 it will need to return an instance of `ActiveSupport::Duration` instead of an instance of `Float`.

```
=> 0.75 seconds
```